chalice/deployer.py

```diff
@@ -163,10 +164,13 @@ class Deployer(object):
     LAMBDA_CREATE_ATTEMPTS = 5
     DELAY_TIME = 3
 
-    def __init__(self, session=None, prompter=None):
+    def __init__(self, session=None, prompter=None, config=None):
```

Comment:
It might be simpler if we just add a `profile` param to the `__init__` for now.

We can always switch this to `config` if we end up needing more things in the `__init__`.